parameters/world_modeling/world_model.yaml

```diff
@@ -94,4 +93,4 @@ plugins:
   lib: libed_tf_publisher_plugin.so
   parameters:
       root_frame_id: /map
-      exclude:
+      exclude: hero
```

Comment:
Idem, because state publisher is publishing this?

Reply:
Yep